ui/src/literature/components/SupervisorList.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+
+import InlineList from '../../common/components/InlineList';
+import Author from '../../common/components/Author';
+import pluralizeUnlessSingle from '../../common/utils';
+
+function renderSupervisor(supervisor) {
+  return <Author author={supervisor} />;
+}
+
+function extractKeyFromSupervisor(supervisor) {
+  return supervisor.get('uuid');
```

Comment:
this is isn't present when author is not linked.

Reply:
It is, we are generating it at lit update for every author (linked or unlinked - we use it for the disambiguation to identify signatures).